swarm/storage/localstore.go

```diff
@@ -85,6 +96,13 @@ func (self *LocalStore) CacheCounter() uint64 {
 // LocalStore is itself a chunk store
 // unsafe, in that the data is not integrity checked
 func (self *LocalStore) Put(chunk *Chunk) {
+	for _, v := range self.Validators {
+		if !v.Validate(chunk.Key, chunk.SData) {
+			chunk.SetErrored(ChunkErrInvalid)
```

Comment:
??? if one fails second can succeed; it is OR ,not AND

Reply:
Whoa, old code. Sorry.